app/javascript/application.js

```diff
@@ -1,3 +1,16 @@
-// Entry point for the build script in your package.json
 import "@hotwired/turbo-rails"
 import "./controllers"
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import store from './redux/store'; 
+
+import Greeting from './components/greeting';
+
+ReactDOM.render(
+  <Provider store={store}>
+    <Greeting />
+  </Provider>,
+  document.getElementById('root')
+);
```

Comment:
- [ ] Great work implementing the Greeting component, everything works as it intend to :clap: However, it requested in the project requirement to create `App` component with `react-router` where the  `Greeting` component will be set up as a route. Therefore I suggest that you create the `App` component and use react-router to create route that link to the `Greeting`.
![Screenshot from 2023-12-08 05-26-07](https://github.com/mershark/hello-rails-react/assets/91142432/ff5623f1-c060-4c85-b65a-5f4abf94186f)